apis/generate.go

```diff
@@ -0,0 +1,33 @@
+// +build generate
```

Comment:
Do you think `./tools.go` would be a more generic and appropriate name for this file?

Reply:
I thought about that, but I went with `generate.go` because this file contains our code generation tags as well as the tool imports required to power those tags. Most of the examples that use `tools.go` seem to be basing the file name on the fact that it _only_ handles the imports for tools that are used elsewhere.